src/menus.jrl

```diff
@@ -271,7 +271,7 @@ p({"class":"foam.core.menu.Menu","id":"reset","label":"Reset Password","handler"
 p({
   "class":"foam.core.menu.Menu",
   "id":"sign-in",
-  "authenticate":false,
+  "authorizationStatus":1,
```

Comment:
Can we specify journal entries as the enum value, like: 'UNAUTHENTICATED'? If so, that would make journals a lot more readable.